### PR TITLE
[Fix] [Advanced search]: Properties containing space at the end of Label are not found

### PR DESCRIPTION
**Pull request by habemuscode**
**Ticket**

https://oat-sa.atlassian.net/browse/ADF-475

### CODE REVIEW

**Comment by AntonNeviarouski** [comment]
QA verified on http://aadvanced-search-integration.playground.kitchen.it.taocloud.org:48321/

File: views/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "@oat-sa/tao-core-libs": "0.4.3",
         "@oat-sa/tao-core-sdk": "1.15.0",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.27.5",
+        "@oat-sa/tao-core-ui": "git://github.com/oat-sa/tao-core-ui-fe#fix/ADF-475/properties-with-end-space-not-found",
```
Comment (404passionFruit) [inline views/package.json:21]: Just a comment to not forget update to the proper version

**Review by 404passionFruit — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`
- [x] Pull request title and description are meaningful
- [x] Commits are following conventional commits
- [x] Kitchen links are not exposed in the description

**Review by clagosarias — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  The feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request's target is not master
- [x]  Pull request title and description are meaningful
- [x]  Commits are following conventional commits
- [x]  Kitchen links are not exposed in the description

File: views/package.json (hunk 1 above)
Comment (habemuscode) [reply views/package.json:21]: Done

**Pull request merged**